src/leaky_relu.js

```diff
@@ -9,12 +9,6 @@ import {Tensor} from './tensor.js';
  * @return {Tensor}
  */
 export function leakyRelu(input, options = {}) {
-  const alpha = options.alpha ? options.alpha : 0.01;
-  const output = new Tensor(input.shape);
-  for (let i = 0; i < input.size; ++i) {
-    const x = input.getValueByIndex(i);
-    const y = Math.max(0, x) + alpha * Math.min(0, x);
-    output.setValueByIndex(i, y);
-  }
-  return output;
+  const alpha = options.alpha !== undefined ? options.alpha : 0.01;
```

Comment:
Here I updated the condition code, when `options.alpha` is set as `0`, the `leayRelu` actually acts `relu`
```
max(0, x) + alpha ∗ min(0, x)
```
, while the return value of `options.alpha ? options.alpha : 0.01` is `0.01`.

Reply:
Good catch. We should use `options.alpha !== undefined` for all the cases.